Pages/FoodPages/Edit.cshtml.cs

```diff
@@ -37,14 +37,20 @@ public EditModel(FoodKept.Data.ShopContext context)
         [BindProperty]
         public List<SelectListItem> EnumCategories { get; set; }
 
-        public async Task<IActionResult> OnGetAsync(int? id)
+        public IActionResult OnGet(int? id)
```

Comment:
This method was changed because of lazy initialization?

Reply:
Yes, await methods was not used so async function is not necessary.